scripts/ci/build-version.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Should this be a shared script in the build container?

Reply:
Sure, seems generic enough to be useful across projects